pageserver/src/walredo/apply_neon.rs

```diff
@@ -230,6 +232,14 @@ pub(crate) fn apply_in_neon(
                 LittleEndian::write_u32(&mut page[memberoff..memberoff + 4], member.xid);
             }
         }
+        NeonWalRecord::AuxFile { key, value } => {
+            let mut dir = AuxFilesDirectory::des(page)?;
+            dir.upsert(key.clone(), value.clone());
+
+            page.clear();
```

Comment:
This is of course not particularly efficient, but tenants using a reasonably small stack of deltas shouldn't care.  Tenants with lots of deltas to their slots will experience the cost of this in their basebackup latency.